fiona/fio/cat.py

```diff
@@ -18,7 +18,7 @@
 
 # Cat command
 @click.command(short_help="Concatenate and print the features of datasets")
-@cligj.files_in_arg
```

Comment:
This uses `click.Path()` which chokes on the VFS paths, so I copied the source and removed the type declaration.